core/transport/src/grpc/version.proto

```diff
@@ -8,20 +8,21 @@ package alluxio.grpc.version;
 
 
 enum ServiceType {
-  FILE_SYSTEM_MASTER_CLIENT_SERVICE = 0;
-  FILE_SYSTEM_MASTER_WORKER_SERVICE = 1;
-  FILE_SYSTEM_MASTER_JOB_SERVICE = 2;
-  BLOCK_MASTER_CLIENT_SERVICE = 3;
-  BLOCK_MASTER_WORKER_SERVICE = 4;
-  META_MASTER_CONFIG_SERVICE = 5;
-  META_MASTER_CLIENT_SERVICE = 6;
-  META_MASTER_MASTER_SERVICE = 7;
-  METRICS_MASTER_CLIENT_SERVICE = 8;
-  JOB_MASTER_CLIENT_SERVICE = 9;
-  JOB_MASTER_WORKER_SERVICE = 10;
-  KEY_VALUE_MASTER_CLIENT_SERVICE = 11;
-  KEY_VALUE_WORKER_SERVICE = 12;
-  UNKNOWN_SERVICE = 13;
+  UNKNOWN_SERVICE = 0;
+  FILE_SYSTEM_MASTER_CLIENT_SERVICE = 1;
+  FILE_SYSTEM_MASTER_WORKER_SERVICE = 2;
+  FILE_SYSTEM_MASTER_JOB_SERVICE = 3;
+  BLOCK_MASTER_CLIENT_SERVICE = 4;
+  BLOCK_MASTER_WORKER_SERVICE = 5;
+  META_MASTER_CONFIG_SERVICE = 6;
+  META_MASTER_CLIENT_SERVICE = 7;
+  META_MASTER_MASTER_SERVICE = 8;
+  METRICS_MASTER_CLIENT_SERVICE = 9;
+  JOB_MASTER_CLIENT_SERVICE = 10;
+  JOB_MASTER_WORKER_SERVICE = 11;
+  KEY_VALUE_MASTER_CLIENT_SERVICE = 12;
```

Comment:
Since we've removed KV from alluxio, can the `KEY_VALUE_*` services also be removed?

Reply:
Nice catch.